elasticsearch/includes/classes/commands/HealthCommand.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Automattic\VIP\Elasticsearch;
```

Comment:
Should the namespace be `Automattic\VIP\Elasticsearch\Commands`?

Reply:
Ooops. c88f5462